src/libraries/System.ComponentModel.TypeConverter/tests/CustomTypeDescriptorTests.cs

```diff
@@ -25,7 +25,7 @@ public static IEnumerable<object[]> GetAttributes_TestData()
             yield return new object[] { new AttributeCollection(new EditorBrowsableAttribute()) };
         }
 
-        [Theory]
+        [ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.IsNotMonoAOT))] // Mock will try to JIT 
```

Comment:
I think this is rather about RefEmit not just JIT right? we should unify the conditional to that, this PR seems to be using both.